src/main/java/com/mixpanel/android/viewcrawler/ViewCrawler.java

```diff
@@ -518,38 +534,33 @@ private void sendDeviceInfo() {
                         j.name(entry.getKey()).value(entry.getValue());
                     }
 
-                    final Map<String, Tweaks.TweakDescription> tweakDescs = mTweaks.getDescriptions();
+                    final Map<String, Tweaks.TweakValue> tweakDescs = mTweaks.getAllValues();
                     j.name("tweaks").beginArray();
-                    for (Map.Entry<String, Tweaks.TweakDescription> tweak:tweakDescs.entrySet()) {
-                        final Tweaks.TweakDescription desc = tweak.getValue();
+                    for (Map.Entry<String, Tweaks.TweakValue> tweak:tweakDescs.entrySet()) {
+                        final Tweaks.TweakValue desc = tweak.getValue();
                         final String tweakName = tweak.getKey();
                         j.beginObject();
                         j.name("name").value(tweakName);
-                        j.name("minimum").value(desc.minimum);
-                        j.name("maximum").value(desc.maximum);
+                        j.name("minimum").value((Number) null);
+                        j.name("maximum").value((Number) null);
```

Comment:
We don't support them in the api right now, so they'll always be null anyway, and I wanted to reduce the public API of `Tweaks` to the smallest thing I could before the launch.